sphinx/tutorial/cylc/furthertopics/inheritance.rst

```diff
@@ -0,0 +1,603 @@
+   AIR_VEHICLE
+   AIRPLANE     # sets "CAN_TAKE_OFF_VERTICALLY to "false"
+   HELICOPTER   # sets "CAN_TAKE_OFF_VERTICALLY to "true"
+   v22
+
+Inspect the configuration of the ``v22`` task using ``cylc get-config`` to
+confirm this.
+
+
+More Inheritance
+----------------
+
+We will now add some more families and tasks to the suite.
+
+Engine Type
+^^^^^^^^^^^
+
+Next we will define three families to represent different types of engine.
+
+.. digraph:: Example
+   :align: center
+
+   size = "5,5"
+   bgcolor=none
+
+   ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   TURBINE_ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   INTERNAL_COMBUSTION_ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+   HUMAN_ENGINE [color=royalblue, fillcolor=powderblue, shape=box, style=filled]
+
+   ENGINE -> TURBINE_ENGINE
+   ENGINE -> INTERNAL_COMBUSTION_ENGINE
+   ENGINE -> HUMAN_ENGINE
+
+Each engine type should set an environment variable called ``FUEL`` set to the
+following values:
+
+* Turbine - kerosene
+* Internal Combustion - petrol
+* Human - pizza
+
+Add lines to the ``runtime`` section to represent these four families.
+
+.. spoiler:: Solution warning
+
+   .. code-block:: cylc
+
+          [[ENGINE]]
+          [[TURBINE_ENGINE]]
+              inherit = ENGINE
+              [[[environment]]]
+                  FUEL = kerosene
+          [[INTERNAL_COMBUSTION_ENGINE]]
+              inherit = ENGINE
+              [[[environment]]]
+                  FUEL = petrol
+          [[HUMAN_ENGINE]]
+              inherit = ENGINE
+             [[[environment]]]
+                  FUEL = pizza
+
+We now need to make the three aircraft inherit from one of the three engine
+families. The aircraft use the following types of engine:
+
+* A380 - turbine
+* R44 - internal combustion
+* V22 - turbine
+
+Modify the three tasks so that they inherit from the relevant engine families.
+
+.. spoiler:: Solution warning
+
+   .. code-block:: cylc
+
+         [[a380]]
+             inherit = AIRPLANE, TURBINE_ENGINE
+             [[[meta]]]
+                 title = Airbus A380 Jumbo-Jet
+         [[r44]]
+             inherit = HELICOPTER, INTERNAL_COMBUSTION_ENGINE
+             [[[meta]]]
+                 title = Robson R44 Helicopter
+         [[v22]]
+             inherit = AIRPLANE, HELICOPTER, TURBINE_ENGINE
+             [[[meta]]]
+                 title = V-22 Ofsprey Military Aircraft
+
+Penny Farthing
+^^^^^^^^^^^^^^
+
+Next we want to add a new type of vehicle, an old-fashioned bicycle called a
+penny farthing.
+
+.. image:: https://upload.wikimedia.org/wikipedia/commons/a/a7/Ordinary_bicycle01.jpg
+   :width: 300px
+   :alt: Penny Farthing Bicycle
+   :align: center
+
+To do this we will need to add a new classification of vehicle ``LAND_VEHICLE``,
+and a new subclass of vehicle called ``BICYCLE`` and a new task called
+``penny_farthing`` related in the following manner:
```

Comment:
This would be the best time to remind the reader about capitalisation RE families and tasks - I would move the reminder box on lines 73-77 to here personally.